http-ballerina/src/http/client_endpoint.bal

```diff
@@ -253,7 +253,10 @@ public client class Client {
     public remote function getResponse(HttpFuture httpFuture) returns Response|ClientError {
         Response|ClientError response = self.httpClient->getResponse(httpFuture);
         if (response is Response) {
```

Comment:
Let's check whether observability is enabled here using `isObservabilityEnabled` final variable in this module